src/main/java/com/google/devtools/build/lib/remote/RemoteOptions.java

```diff
@@ -30,11 +38,54 @@
     documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
     effectTags = {OptionEffectTag.UNKNOWN},
     help =
-        "A base URL of a HTTP caching service. Both http:// and https:// are supported. BLOBs are "
+        "A base URL of a HTTP caching service. http://, https:// and s3:// are supported. BLOBs are "
```

Comment:
At `remote-caching.md:195` (above), you say that s3:// urls are not supported, but this seems to imply otherwise...?

Reply:
I think we should not officially support s3:// URLs but stick to the HTTP caching protocol.